.ci/config/stage-types.yaml

```diff
@@ -65,17 +70,31 @@ old-engine:
   - 'mariadb'
   - 'sqlserver'
   - 'postgresql'
+  podSpec:
+    cpu: 4
+    images: ['maven:3.9.7-eclipse-temurin-11', 'postgres:17.0']
 rolling-update:
   directory: 'qa/test-db-rolling-update'
   command: 'verify -Prolling-update,'
   stash:
     runtimeStash: true
     qaStash: true
+  labels:
+    - 'postgresql'
```

Comment:
**Context**: This label has been added to enable the postgres db to be added in the calculated stages of `rolling-update`.
**Why**: It does not happen on PR builds, only enabled on `master` builds.

**TODO:** Remove once testing is done.

Reply:
- Testing of stage is done (see [breakdown](https://github.com/camunda/camunda-bpm-platform/pull/4552#:~:text=rolling%2Dupdate%2Dpostgresql%2D170%20(ref)))
- Removed line ([commit](https://github.com/camunda/camunda-bpm-platform/pull/4552/commits/2d19cd30a3def5c7f3779d86f3d1aafdf4a93a98))